### PR TITLE
Add backgroundContainerError and textError tokens

### PR DESCRIPTION
**Pull request by paulasuarezb**
#1708 Rowlist danger

### CODE REVIEW

File: tokens/blau.json [hunk 1]
```diff
@@ -25,6 +25,11 @@
       "type": "color",
       "description": "white"
     },
+    "backgroundContainerError": {
+      "value": "{palette.white}",
```
Comment (yceballost) [inline tokens/blau.json]: sorry, I can't understand why `white` is set up for this token for all brands. Shouldn't it be equivalent to `errorLow`?

image

This comment apply to all skins

**Review by yceballost — CHANGES_REQUESTED** [review]

**Comment by yceballost** [comment]
Please, change the PR name to something more specific. Like "Add backgroundContainerError and textError tokens"

**Review by aweell — COMMENTED** [review]
I'm missing `textErrorInverse` token

File: tokens/blau.json (hunk 1 above)
Comment (paulasuarezb) [reply tokens/blau.json]: should be the same color in backgroundContainerError and errorLow, you're right! I'll change it

File: tokens/movistar.json [hunk 2]
```diff
@@ -1093,6 +1113,16 @@
       "type": "color",
       "description": "pepper"
     },
+    "textError": {
+      "value": "{palette.pepper}",
+      "type": "color",
+      "description": "pepper"
+    },
+    "textErrorInverse": {
```
Comment (aweell) [inline tokens/movistar.json:1121]: `textErrorInverse` in dark should be = `textError` in all brands

**Review by aweell — CHANGES_REQUESTED** [review]

File: tokens/blau.json (hunk 1 above)
Comment (yceballost) [reply tokens/blau.json]: You didn't change it! xD

I think you make changes in dark mode and not in light mode

File: tokens/blau.json [hunk 3]
```diff
@@ -668,6 +683,11 @@
       "type": "color",
       "description": "darkModeGrey"
     },
+    "backgroundContainerError": {
+      "value": "{palette.blauRed10}",
```
Comment (yceballost) [inline tokens/blau.json]:
```suggestion
      "value": "{palette.darkModeGrey}",
```
Here the right color should be the same as `backgroundContainer` in dark mode, i.e. darkModeGrey

**Review by yceballost — CHANGES_REQUESTED** [review]

**Comment by aweell** [comment]
You can check the effect of your changes here: https://mistica-design.vercel.app/tokens-map?branch=%231708-Rowlist-danger&skin=movistar&tokenType=color&activeColor=undefined

File: tokens/blau.json (hunk 3 above)
Comment (aweell) [reply tokens/blau.json]: +1

* All `backgroundContainerError` tokens in **light** should have the same value that `errorLow`
* All `backgroundContainerError` tokens in **dark** should have the same value that `backgroundContainer`

**Comment by paulasuarezb** [comment]
I think I got it now 😅 that url was really helpful

**Review by yceballost — APPROVED** [review]

File: tokens/o2-new.json [hunk 4]
```diff
@@ -482,6 +487,16 @@
       "type": "color",
       "description": "o2Red65"
     },
+    "textError": {
+      "value": "{palette.o2Red45}",
```
Comment (aweell) [inline tokens/o2-new.json]:
```suggestion
      "value": "{palette.o2Red65}",
```

File: tokens/o2-new.json [hunk 5]
```diff
@@ -482,6 +487,16 @@
...
+      "type": "color",
+      "description": "o2Red45"
```
Comment (aweell) [inline tokens/o2-new.json]:
```suggestion
      "description": "o2Red65"
```

File: tokens/vivo-new.json [hunk 6]
```diff
@@ -450,6 +455,16 @@
       "type": "color",
       "description": "pepper"
     },
+    "textError": {
+      "value": "{palette.pepper}",
+      "type": "color",
+      "description": "pepper"
+    },
+    "textErrorInverse": {
+      "value": "{palette.pepper}",
```
Comment (aweell) [inline tokens/vivo-new.json]:
```suggestion
      "value": "{palette.white}",
```

File: tokens/vivo-new.json [hunk 7]
```diff
@@ -450,6 +455,16 @@
...
+      "type": "color",
+      "description": "pepper"
```
Comment (aweell) [inline tokens/vivo-new.json]:
```suggestion
      "description": "white"
```

**Review by aweell — CHANGES_REQUESTED** [review]

File: tokens/vivo-new.json (hunk 6 above)
Comment (yceballost) [reply tokens/vivo-new.json]: why white? in dark mode we can use red in inverse 🤔

Comment (aweell) [reply tokens/vivo-new.json]: This is light mode in dark mode is pepper which is correct

**Review by aweell — APPROVED** [review]

**Pull request merged**